File: examples/python/Calculator.py
import json

def handler(event, context):
    num1 = float(event.get('num1', 0))
    num2 = float(event.get('num2', 1))

    addition = num1 + num2
    subtraction = num1 - num2
    multiplication = num1 * num2
    division = num1 / num2 if num2 != 0 else 'undefined'

    # Return the results
    return {
        'statusCode': 200,
        'body': json.dumps({
            'addition': addition,
            'subtraction': subtraction,
            'multiplication': multiplication,
            'division': division
        })
    }

File: examples/nodejs/src/Calculator.mjs
export const handler = async (event) => {
  const num1 = parseFloat(event.num1) || 0;
  const num2 = parseFloat(event.num2) || 1;

  const addition = num1 + num2;
  const subtraction = num1 - num2;
  const multiplication = num1 * num2;
  const division = num2 !== 0 ? num1 / num2 : 'undefined';

  // Return the results
  const response = {
      statusCode: 200,
      body: JSON.stringify({
          addition: addition,
          subtraction: subtraction,
          multiplication: multiplication,
          division: division
      }),
  };
  return response;
};
